specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/ServerConfigurationOptions.json

```diff
@@ -0,0 +1,264 @@
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Updates managed instance server configuration option.": {
+            "$ref": "./examples/ServerConfigurationOptionUpdate.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ServerConfigurationOption": {
+      "description": "A server configuration option",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ServerConfigurationOptionProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ServerConfigurationOptionListResult": {
+      "description": "A list of server configuration options.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ServerConfigurationOption"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ServerConfigurationOptionProperties": {
+      "description": "The properties of server configuration option.",
+      "required": [
+        "serverConfigurationOptionValue"
+      ],
+      "type": "object",
+      "properties": {
+        "serverConfigurationOptionValue": {
```

Comment:
>serverConfigurationOptionValue [](http://example.com/codeflow?start=9&length=30)

should this only have update and read mutability if its not creatable?

Reply:
I will add update and read mutability

Reply:
Added